client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -480,6 +490,86 @@ body {
   background-color: fade(@redash-gray, 12%) !important;
 }
 
+.refresh-indicator {
```

Comment:
Something that I missed when doing the review: we need to stop adding styles to `redash-newstyle.less`. The refresh indicator should've been a component and its style should've been next to it.

Not worth revisiting at the moment, just wanted to point out so we keep this in mind for future stuff.

Reply:
Had to be done! https://github.com/getredash/redash/pull/4017